packages/admin/cms-admin/src/generator/future/generateForm/generateFields.ts

```diff
@@ -10,6 +10,12 @@ export type GenerateFieldsReturn = GeneratorReturn & {
     hooksCode: string;
     formFragmentFields: string[];
     formValueToGqlInputCode: string;
+    formValuesConfig: {
+        omitFromFragmentType?: string;
+        typeLines: string[];
+        initializationLines: string[];
+        defaultInitializationLines: string[];
```

Comment:
why lines and not a code string? (to be similar to existing api)

Reply:
that's a good idea. It's code not lines, you're right.